api/base/filters.py

```diff
@@ -284,7 +284,7 @@ def get_query_from_request(self):
         if self.request.parser_context['kwargs'].get('is_embedded'):
             param_query = None
         else:
-            param_query = self.query_params_to_odm_query(self.request.QUERY_PARAMS)
+            param_query = self.query_params_to_odm_query(self.request.query_params)
```

Comment:
```
ipdb> request.QUERY_PARAMS
*** NotImplementedError: `request.QUERY_PARAMS` has been deprecated in favor of `request.query_params` since version 3.0, and has been fully removed as of version 3.2.
```
:+1: